src/service/bahmni-encounter.js

```diff
@@ -0,0 +1,46 @@
+import { dispenseConceptURL, getRequest, postRequest, sessionURL } from "../utils/api-utils";
+
+const bahmniEncounterPost = async (data, location) => {
+	const dispenseConceptData = await getRequest(dispenseConceptURL);
+	const session = await getRequest(sessionURL)
+	const observations = data.dispense_drugs.map((drug) => ({
+		providers: [
+			{
```

Comment:
Why is this needed? Provider information is not needed at observation level right ?

Reply:
yeah, observation doesn't require providers details.. will remove it

Reply:
removed